### DISCUSSION TITLE
part-ii/2019-7/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#57] [Question comments]
Part II 2019 Algebraic Topology: Paper 1, Section II, F 

https://questions.tripos.org/part-ii/2019-7/

**Comment by eha1234** [comment]
For part (c), I got that no such space exists.

**Comment by Kiolt** [comment]
<details>
<summary>Part (b).</summary>

If n is odd take n points in a row with loops at the end. If n is even add a copy of the wedge outside the row.

</details>

<details>
<summary>Part (c).</summary>

No, because the corresponding group is index 2 so normal. From here use the GLP to lift the covering map to a covering transformation which swaps preimages of some basepoint.

</details>

**Comment by ArthurConmy** [comment]
I believe that a different way of seeing c) (though probably under the hood the same as @Kiolts) is that degree 2 covering maps correspond under the Galois correspondence to index 2 subgroups which are all normal, and hence the associated group of deck transformations exists and is of order 2. The non-trivial element is not identity, because it does not fix the basepoint.
